### ISSUE TITLE
Unexpected error passing apiOptions

### ISSUE THREAD

**Issue by DEV-ONI** [storyblok/storyblok-nuxt#102]
I seem to be getting an error trying to pass options through the `apiOptions` config in `buildModules`

Using this:

```js
[
      "@storyblok/nuxt/module", 
      { 
        accessToken: process.env.STORYBLOK_PREVIEW_API_KEY,
        apiOptions: {
          cache: 'memory'
        }, // storyblok-js-client options
        bridge: true,
        useApiClient: true
      }
    ],
]
```

I seem to get this.
Removing `apiOptions` clears the error but I need to pass config to the js-client. 

Syntax Error: Unexpected token (14:17)                                                                                                                                    friendly-errors 15:36:34

  12 |     accessToken: "8OooEHvfrdAHQAAokOArXgtt",
  13 |     bridge: true,
\> 14 |     apiOptions: {&quot;cache&quot;:&quot;memory&quot;},
     |                  ^
  15 |
  16 |     use: [apiPlugin]
  17 |   });
  
  Any idea what is causing this?
  It seems to be a problem with stringifying the `apiOptions` object

**Comment by philipp-bender** [comment]
I have the same issue and there is no way pass in the data.
The plugin is using the HTML "escape" delimiter to escape data property values:

`apiOptions: <%- JSON.stringify(options.apiOptions || {}) %>,`

and this results into this compiled code:

`apiOptions: {&quot;endpoint&quot;:&quot;/api/storyblok&quot;},`

So please remove this escape delimiter.

**Comment by venux92** [comment]
Same issue, can't pass any options
Any ideas when this will be resolved?

**Comment by alexjoverm** [comment]
Hi all! FYI - we moved Nuxt 2 module to https://github.com/storyblok/storyblok-nuxt-2

I've just fixed this issue in the latest commit https://github.com/storyblok/storyblok-nuxt-2/commit/6d842b5eee227af8d0f31e951841838002ec9a12.

Released in `@storyblok/nuxt-2` v1.0.1
